src/main/java/com/spotify/confidence/ConfidenceValue.java

```diff
@@ -320,24 +352,24 @@ public boolean isDate() {
 
     @Override
     public LocalDate asLocalDate() {
-      return value;
+      return super.value;
     }
 
     @Override
     public String toString() {
-      return String.valueOf(value);
+      return String.valueOf(super.value);
     }
 
     @Override
     public com.google.protobuf.Value toProto() {
-      return com.google.protobuf.Value.newBuilder().setStringValue(value.toString()).build();
+      return com.google.protobuf.Value.newBuilder().setStringValue(super.value.toString()).build();
     }
   }
 
-  public static class List extends ConfidenceValue {
-    private final ImmutableList<ConfidenceValue> values;
+  public static class List<T> extends ConfidenceValue {
```

Comment:
Note to self: look into extending `ConfidenceValue<List<T>>` instead. 
Maybe something applicable to Struct too.